src/components/tooltip/tooltip.test.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/button-has-type */
+import * as React from "react";
+import { forwardRef } from "react";
+// import Button from "../button";
+import Tooltip from "./tooltip.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+import { tooltipPreview } from "../../../cypress/locators/tooltip/index";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+
+const TooltipComponent = ({ ...props }) => {
+  // const [position, setPosition] = useState("top");
+  const Component = forwardRef(({ children }, ref) => (
```

Comment:
do we need this ? imported directly from `react`?

Reply:
I have tried to import Component from react but didn't work for me.